.github/workflows/kakarot_rpc.yml

```diff
@@ -109,3 +109,67 @@ jobs:
       - name: Inspect image
         run: |
           docker buildx imagetools inspect ${{ env.REGISTRY_IMAGE }}:latest
+
+  build-hive:
+    runs-on: ubuntu-latest
```

Comment:
is this powerful enough? how long does the hive build take? should we build on 16 cores?

Reply:
So we're not building the binaries and just taking them from the existing packages, build time is around [7m](https://github.com/alamatxyz/kakarot-rpc/actions/runs/5929518920/job/16077828923)